### PR TITLE
Creation of a button

### LINKED ISSUE #11: Crear componente - button
### Descripción

En la carpeta `components` crear la estructura básica para un componente llamado `button`.

### Criterios de aceptación

- El componente debe contener la estructura básica de un archivo `.vue` (`<template>` `<script>` y `<style>`)
- La etiqueta `template` debe tener una clase con el mismo nombre del componente
- En `script` se le debe asignar al componente el nombre

### Recursos:

- [Vue single file components](https://vuejs.org/v2/guide/single-file-components.html)

### Notas

N/A

### 📆 Fecha limite de entrega:
22/10/2018

### PR DESCRIPTION
**Pull request by andresisazaa**
## Description :Creation of a button:

Creation of a basic structure of a button.

Fixes #11  (issue)

### Type of change :bomb:

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist :white_check_mark:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/button.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="button">
+    <button>
```
Comment (ManuCastrillonM) [inline src/components/button.vue]: por favor borra la etiqueta `button` ya que no hace parte del scope de la historia

File: src/components/button.vue [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      {name}
+    </button>
+  </div> 
+</template>
+
+<script>
+export default {
+    name: "Click me!"
```
Comment (ManuCastrillonM) [inline src/components/button.vue]: Recuerda que `name` debe coincidir con el nombre del componente

File: src/components/button.vue [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+};
```
Comment (ManuCastrillonM) [inline src/components/button.vue]: Segun el estandar del proyecto, no vamos a utilizar `;`

File: src/components/button.vue [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+</script>
+
+<style lang="scss">
+  .button {
```
Comment (ManuCastrillonM) [inline src/components/button.vue]: Por favor no identes esta linea

**Comment by ManuCastrillonM** [comment]
Recuerda que el PR se debe hacer a la rama `develop`, porfa actualizalo

File: src/components/button.vue [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="button">
+    {name}
```
Comment (ManuCastrillonM) [inline src/components/button.vue]: por favor elimina esta linea

**Comment by ManuCastrillonM** [comment]
Recuerda que el PR se debe hacer a la rama `develop`, porfa actualizalo

File: src/components/button.vue (hunk 5 above)
Comment (andresisazaa) [reply src/components/button.vue]: ¿Cuál linea, la 3?

Comment (ManuCastrillonM) [reply src/components/button.vue]: sip

**Comment by MartinEliasQ** [comment]
Borra por favor el archivo package-lock.json para arreglar los conflictos.

**Pull request merged**